web/src/components/atomic/Icon.vue

```diff
@@ -29,34 +30,34 @@
     :path="mdiRadioboxIndeterminateVariant"
     size="1.3rem"
   />
-  <SvgIcon v-else-if="name === 'status-skipped'" :path="mdiMinusCircleOutline" size="1.3rem" />
-  <SvgIcon v-else-if="name === 'status-success'" :path="mdiCheckCircleOutline" size="1.3rem" />
-  <SvgIcon v-else-if="name === 'attention'" :path="mdiAlert" size="1.3rem" />
-  <SvgIcon v-else-if="name === 'warning'" :path="mdiAlertOutline" size="1.3rem" />
-  <SvgIcon v-else-if="name === 'error'" :path="mdiAlertCircle" size="1.3rem" />
-  <SvgIcon v-else-if="name === 'gitlab'" :path="mdiGitlab" size="32" />
+  <SvgIcon v-else-if="name === 'status-skipped'" :path="mdiMinusCircle" size="1.3rem" />
+  <SvgIcon v-else-if="name === 'status-success'" :path="mdiCheckCircle" size="1.3rem" />
+  <SvgIcon v-else-if="name === 'attention'" :path="mdiAlertCircle" size="1.3rem" />
+  <SvgIcon v-else-if="name === 'warning'" :path="mdiAlert" size="1.3rem" />
+  <SvgIcon v-else-if="name === 'error'" :path="mdiAlert" size="1.3rem" />
```

Comment:
Having two icons under different name with the same target `mdiAlert` doesnt make sense. Before that change we had three different icons for attention, warning and error. Why was this changed? How to we differentiate between these 3 now?

Reply:
With color. See OP. We also discussed this earlier in this PR already.

Reply:
Ok sorry. Got confused by the different names.